pkgs/build-support/trivial-builders/default.nix

```
@@ -242,7 +242,14 @@ rec {
 
 
   */
-  writeScriptBin = name: text: writeTextFile {inherit name text; executable = true; destination = "/bin/${name}";};
+  writeScriptBin = name: text: writeTextFile {
+    inherit name text;
+    executable = true;
+    destination = "/bin/${name}";
+    meta = {
+      mainProgram = name;
+    };
```

If there is no other meta attrs,


```suggestion
    meta.mainProgram = name;
```

---

I believe there is no reason to extend this in the future.

Will anyone take `maintainer`ship of these?

They don't run in all `platforms`?

They don't even need `homepage` or `changelog`.

On the other hand, a short `description` is certainly useful, but you didn't bother...